pact/marmalade.repl

```diff
@@ -580,6 +580,82 @@
 
 (commit-tx)
 
+(begin-tx "sale fixed-quote-policy failure")
+(env-hash (hash "fixed-quote-sale-tx2"))
+(use marmalade.ledger)
+(env-chain-data {"block-height": 100})
+(env-sigs
+  [{'key:'bob
+   ,'caps:
+    [(marmalade.ledger.OFFER "project-2" "bob" 0.2 120)]}
+ ])
+
+(env-data
+  {
+   "quote": {
+       "fungible": {
+         "refName": {"namespace": "null", "name": coin},
+         "refSpec": {"namepace": "null", "name": fungible-v2}
+       }
+      ,"price": -1.0
+      ,"recipient": "alice"
+      ,"recipient-guard": {"keys": ["alice"], "pred":"="}
+    }
+  })
+
+(expect-failure
+  "Sale fails with non-fungible in quote"
+  "fungible: Failure: Type error: expected module{fungible-v2}, found object:*"
```

Comment:
This isn't the right thing to test for, that's just invalid data above. Instead test using smth like `kip.manifest` in there as a valid modref to exercise enforcement of being a fungible

Reply:
Changed the test case, but the error message is not as expected. It treats non-fungible modules as objects.

Reply:
Oh I'm sry I was thinking it was JSON not pact. Hmm need to investigate that one further

Reply:
`kip.manifest`, which triggers the namespaced-module-as-modref bug. Try `ns` instead for the test, it doesn't have the issue as a root module.